packages/sdk-client/test/client.test/sdk-client.test.ts

```diff
@@ -8,7 +8,7 @@ import {
 import fetch from 'node-fetch'
```

Comment:
Please add a test that proves your fix is actually working - a test that fails on master branch but passes on your branch.